pkg/flavors/flavors.go

```diff
@@ -50,7 +50,7 @@ func GetFlavor(flavor string) (FlavorConfigTemplate, error) {
 }
 
 // CreateConfig function checks flavor specific parameters, get flavor's template and
-// create a config.
+// create a config yaml used as the `--config-yaml` argument to Envoy.
```

Comment:
new comment says why we don't need any more things except the literal. ps this is only used for postgres I think